backend/api/src/main/kotlin/software/shonk/adapters/incoming/ShorkInterpreterController.kt

```diff
@@ -55,6 +56,17 @@ fun Route.configureShorkInterpreterControllerV1() {
         program.onSuccess { call.respond(Program(it)) }
         return@get
     }
+
+    post("/lobby") {
+        val lobbyId = shorkUseCase.createLobby()
+        val result = shorkUseCase.getLobbyStatus(lobbyId)
+
+        result.onFailure {
```

Comment:
I'd prefer result.getOrThrow here to throw an exception if it is returned, because we can expect the result to be okay. If it's not, that'd be a technical error, not a domain violation

Reply:
agree, and I would like a comment above it explaining exactly that, and why its ok to potentially throw an exception here